plugins/test/unit/plugins/distributors/test_configuration.py

```diff
@@ -140,7 +139,7 @@ def test__is_valid_repo_registry_id(self):
             '134567890',
             'alpha-numeric_123',
             'periods.are.cool',
-            '..............',
+            'repo-registry-id'
```

Comment:
can we add some values with one and more than one slash in here to ensure we're guarded against regression in multi-slash registry ids when we're directly testing the `_is_valid_repo_registry_id` unit?